lib/ansible/modules/storage/netapp/na_ontap_nfs.py

```diff
@@ -254,7 +262,7 @@ def get_nfs_status(self):
         :param:
             name : Name of the vserver
         :return: status of nfs. None if not found.
-        :rtype: bool
+        :rtype: boolean
```

Comment:
?

Reply:
Looks like out internal version didn't have #50010 in it. I'll make the change back to bool.